content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,370 @@
+
+### Pre-configuration
+
+To align memory resources with other requested resources in a Pod Spec:
+- the CPU Manager should be enabled and proper CPU Manager policy should be configured on a Node. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/);
+- the Topology Manager should be enabled and proper Topology Manager policy should be configured on a Node. See [control Topology Management Policies](/docs/tasks/administer-cluster/topology-manager/).
+
+### Enable the Memory Manager feature 
+
+Support for the Memory Manager requires `MemoryManager` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. 
+
+That is, the `kubelet` must be started with the following flag:
+
+`--feature-gates=MemoryManager=true`
+
+### Policies 
+
+Memory Manager supports two policies. You can select a policy via a `kubelet` flag `--memory-manager-policy`.
+
+Two policies can be selected:
+
+* `none` (default)
+* `static`
+
+#### none policy {#policy-none}
+
+This is the default policy and does not affect the memory allocation in any way.
+It acts the same as if the Memory Manager is not present at all.
+
+The `none` policy returns default topology hint. This special hint denotes that Hint Provider (Memory Manger in this case) has no preference for NUMA affinity with any resource.
+
+#### static policy {#policy-static}
+
+In the case of the `Guaranteed` pod, the `static` Memory Manger policy returns topology hints relating to the set of NUMA nodes where the memory can be guaranteed.
+
+In the case of the `BestEffort` or `Burstable` pod, the `static` Memory Manager policy sends back the default topology hint as there is no request for the guaranteed memory.  
+
+### Reserved memory flag
+
+[Node Allocatable Feature](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/) is commonly used by node administrators to reserve K8S node system resources for the kubelet or operating system processes in order to enhance the node stability. A dedicated set of flags can be used for this purpose to set the total amount of reserved memory for a node. This pre-configured value is subsequently utilized to calculate the real amount of node's "allocatable" memory available to pods. Also, K8S scheduler incorporates "allocatable" to optimise pod scheduling process. The foregoing flags include `--kube-reserved`, `--system-reserved` and `--eviction-threshold`. The sum of their values will account for the total amount of reserved memory.    
+
+A new `--reserved-memory` flag was added to Memory Manager to allow for this total reserved memory to be split (by a node administrator) and accordingly reserved across many NUMA nodes. 
+
+Syntax: 
+
+`--reserved-memory=[{numa-node=int,type=string,limit=string}][,][...]`

```

Comment:
the updated format is
```
--reserved-memory 0:memory=1Gi,hugepages-1M=2Gi
--reserved-memory 1:memory=2Gi
```

Reply:
@cynepco3hahue Should not be `1Gi` in the example for hugepages? It's `hugepages-1M`.

Reply:
yep it should be `hugepages-1Gi` or `hugepages-2Mi`